packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -595,18 +624,22 @@ async function startCompass(
   // For webdriverio env we are changing appName so that keychain records do not
   // overlap with anything else
   process.env.HADRON_PRODUCT_NAME_OVERRIDE = 'MongoDB Compass WebdriverIO';
+  // TODO: we should probably do this for compass-web too
   process.env.DEBUG = `${process.env.DEBUG ?? ''},mongodb-compass:main:logging`;
+  // TODO: we should probably do this for compass-web too
   process.env.MONGODB_COMPASS_TEST_LOG_DIR = path.join(LOG_PATH, 'app');
   process.env.CHROME_LOG_FILE = chromedriverLogPath;
 
   // Guide cues might affect too many tests in a way where the auto showing of the cue prevents
   // clicks from working on elements. Dealing with this case-by-case is way too much work, so
   // we disable the cues completely for the e2e tests
+  // TODO: also for compass-web
   process.env.DISABLE_GUIDE_CUES = 'true';
 
   const options = {
     capabilities: {
-      browserName: 'chrome',
+      browserName: 'chromium',
+      browserVersion: '120.0.6099.216', // TODO: this must always be the corresponding chromium version for the electron version
```

Comment:
This kinda sucks. We have to set this otherwise webdriver now goes down a path where it tries to use the electron version as the chromium version and then gets the chrome driver version for that version.. Electron always includes the chromium version in their release notes. Maybe there's a way to programmatically get this? https://releases.electronjs.org/release/v28.1.4